src/utils.ts

```diff
@@ -435,3 +436,14 @@ export async function* combineAsyncIterables<T>(
 	}
 	return;
 }
+
+/**
+ * Converts a path to a unix-style path if absolute windows path and cross-seed is on linux.
+ * @param rawPath The path to convert
+ * @returns The converted path
+ */
+export function upath(rawPath: string): string {
+	if (path.sep === "\\") return rawPath; // cross-seed on windows but linux client can never work, assume client is windows
+	if (ABS_WIN_PATH_REGEX.test(rawPath)) return rawPath.replace(/\\/g, "/");
+	return rawPath;
+}
```

Comment:
is this used?

Reply:
No it was from an old implementation I forgot to remove.